static/app/components/replaysOnboarding/sidebar.tsx

```diff
@@ -65,19 +103,35 @@ function ReplaysOnboardingSidebar(props: CommonSidebarProps) {
       <TopRightBackgroundImage src={HighlightTopRightPattern} />
       <TaskList>
         <Heading>{t('Getting Started with Session Replay')}</Heading>
-        <DropdownMenu
-          items={items}
-          triggerLabel={
-            <StyledIdBadge
-              project={selectedProject}
-              avatarSize={16}
-              hideOverflow
-              disableLink
-            />
-          }
-          triggerProps={{'aria-label': selectedProject.slug}}
-          position="bottom-end"
-        />
+        <div
+          onClick={e => {
+            // we need to stop bubbling the CompactSelect click event
+            // failing to do so will cause the sidebar panel to close
+            // the event.target will be unmounted by the time the panel listener
+            // receives the event and assume the click was outside the panel
+            e.stopPropagation();
```

Comment:
Damn, assuming no onClick on CompactSelect?

Reply:
Oh man, I went down the rabbit hole on this last time around for profiling's onboarding. 

If i recall correctly, the compactSelect renders a portal that is seen as _outside_ of the sidebar, so when you click on a project the sidebar sees it as a click outside and it will close